server/services/rolledOutPrisonService.ts

```diff
@@ -0,0 +1,12 @@
+import { Request, Response } from 'express'
+import ActivitiesService from './activitiesService'
+
```

Comment:
A design question - not sure why the extra service for a single function?  The way you have it is:

healthRoutes -> rolloutPrisonService -> activitiesService -> activitiesApiClient -> gets data

Would it be simpler to add the activeRolledOutPrisons() function into the activitiesService and have one less level?

healthRoutes -> activitiesService.getRolledOutPrisons() -> activitiesApiClient.getRolledOutPrisons() -> get data.